site_scons/prereq_tools/base.py

```diff
@@ -957,19 +957,24 @@ def add_options(self):
     def __parse_build_deps(self):
         """Parse the build dependances command line flag"""
         build_deps = GetOption('build_deps')
-        if build_deps == 'yes':
+        if build_deps == 'yes' or build_deps == 'only':
```

Comment:
(pylint-consider-using-in) Consider merging these comparisons with "in" to "build_deps in ('yes', 'only')"